pkg/cluster/install.go

```diff
@@ -46,11 +46,7 @@ func (m *manager) adminUpdate() []steps.Step {
 		steps.Action(m.initializeKubernetesClients), // must be first
 		steps.Action(m.ensureBillingRecord),         // belt and braces
 		steps.Action(m.ensureDefaults),
-
-		// TODO: this relies on an authorizer that isn't exposed in the manager
-		// struct, so we'll rebuild the fpAuthorizer and use the error catching
-		// to advance
-		steps.AuthorizationRetryingAction(m.fpAuthorizer, m.fixupClusterSPObjectID),
```

Comment:
This comment has changed: https://github.com/azure/ARO-RP/blob/176a68d4b8494cbbdf696334a36823c1c468d490/pkg/cluster/install.go#L197

Is this change still good?